Elmah.Io.ElasticSearch/ElasticClientSingleton.cs

```diff
@@ -18,9 +18,13 @@ public class ElasticClientSingleton : IDisposable
         private static ElasticClientSingleton _instance;
         public IElasticClient Client;
 
-        private ElasticClientSingleton()
+        private ElasticClientSingleton(IDictionary config = null)

```

Comment:
@TrymBeast with your update, would there ever be a scenario where this would be passed in as null?  I'm thinking it could be mandatory.